options/flags.go

```diff
@@ -73,7 +73,7 @@ func (o *Options) AddFlags(cmd *cobra.Command) {
 		&o.Repo,
 		FlagRepo,
 		o.Repo,
-		"repository to check",
+		"URL of repository to check",
```

Comment:
Hmmm, maybe we should specify the expected form here instead?

```suggestion
		"repository to check (valid inputs: "owner/repo", "github.com/owner/repo")",
```

Reply:
Done.